docs/baseplate/core.rst

```diff
@@ -58,6 +58,37 @@ integrations <integration/index>`.
 .. autoclass:: TraceInfo
    :members: from_upstream
 
+Authentication
+--------------
+
+If the application is dealing with authentication either directly from the
+authentication service or from upstream services, there are some support interfaces
+available for easily working with passing context down or picking up passed down
+contexts.
+
+When receiving an initial authentication token from the authentication service, the
+application should properly store the token in an
+:py:class:`~baseplate.core.AuthenticationContext` object attached to the current
```

Comment:
I had thought about adding some classmethod to do the construction and binding, but also wanted to give something more explicit for how the binding looks.  I will ponder this some more...